src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import cn from 'classnames';
+import { SearchLink } from '../SearchLink';
+import { SexFilter } from '../../types/SexFilter';
+
+type Props = {
+  query: string;
+  changeQuery: (input: string) => void;
+  sex: string | null;
+  centuries: string[];
+};
+
+const sexCollection: SexFilter[] = [
+  { id: 0, sex: null, title: 'All' },
+  { id: 1, sex: 'm', title: 'Male' },
+  { id: 2, sex: 'f', title: 'Female' },
+];
+
+const centuryCollection = ['16', '17', '18', '19', '20'];
+
+export const PeopleFilters: React.FC<Props> = ({
+  query,
+  changeQuery,
+  sex,
+  centuries,
+}) => {
+  const changeCentury = (century: string) => (centuries.includes(century)
+    ? centuries.filter(currentCentury => currentCentury !== century)
+    : [...centuries, century]);
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    changeQuery(event.target.value);
+  };
+
+  return (
+    <nav className="panel">
+      <p className="panel-heading">Filters</p>
+
+      <p className="panel-tabs" data-cy="SexFilter">
+        {sexCollection.map(sexItem => (
+          <SearchLink
+            key={sexItem.id}
+            className={cn({
```

Comment:
To improve the readability of the code, it is better to transfer the definition of classes to separate variables and use the ready-made result. You can see an example of use on the website of the [classnames ](https://www.npmjs.com/package/classnames)package.